Fixing compatibility with Python 3.5.2 and updating Readme

madnihamza1841 opened the pull request:
**Description:**
Fixing Python compatibility issues for Python 3.5.2 and beyond. Also updated ReadMe to exclude i18n documentation.

**JIRA:**
https://edlyio.atlassian.net/browse/EDLY-4788

ziafazal commented:
@madnihamza1841 could you please take care of couple of things

1. Version bump [here](https://github.com/edly-io/instruqt-xblock/blob/master/instruqtxblock/__init__.py#L4)
2. Format commit message according to these [guidelines](https://github.com/openedx/open-edx-proposals/blob/1b1edfdea09095bde09cda17e72fdc7f5e818376/oeps/oep-0051-bp-conventional-commits.rst)

madnihamza1841 committed:
fix: Python 3.5.2 Compatibility issues and Update ReadMe
    The package was just working with Python>3.8:

    -  was updated for python 3.5 support

    - F-strings were removed from code and .format was used in those places.

    Fixes: EDLY-4788

ziafazal reviewed (approved):
👍
